README.md

```diff
@@ -81,294 +81,93 @@ You surely want to combine it with running [Xvfb](https://gist.github.com/nwinkl
-
-You can skip testing specfic rules with the **skipTest** configuration. In this example we skip the CDN rule and the textcontent rule ([here](http://www.sitespeed.io/rules/) is the rule list, you find the rule id inside the parentheses per rule.
-
-```javascript
-{
-	urls: ["http://www.sitespeed.io", "http://www.sitespeed.io/faq/"],
-	skipTest: "ycdn,textcontent",
-	budget: {
-		rules: {
-			default: 90
-		}
-	}
-}
-```
-#### Performance Budget page metrics
-Keep track of the page metrics of your site like number of requests per page, page weight, number of requests per type and weight per type.
-
-You can configure the following
-* requests - number of requests
-* pageWeight - the total page weight
-* js and jsWeight
-* css and cssWeight
-* image and imageWeight
-* cssimage and cssimageWeight
-* font and fontWeight
-* flash and flashWeight
-* iframe
-* doc and docWeight
-
-```javascript
-{
-	urls: ["http://www.sitespeed.io", "http://www.sitespeed.io/faq/"],
-	headless: 'slimerjs',
-	budget: {
-		page: {
-			image: 10,
-			css: 1,
-			requests: 7,
-			pageWeight:300000,
-			jsWeight: 0
-		}
-	}
-}
-```
-
-
-#### Performance Budget timings
-You can test your sites browser timings fetched using the Navigation Timing API and the User Timing API. You can test the following [metrics](http://www.sitespeed.io/documentation/#timingMetrics). 
-
-In the follwing example we test the RUM speed index score, headerTime and logoTime (specific user timings implemented on this site) and the domContentLoadedTime. If any timings are bigger than configured, it will fail.
-
-
-```javascript
-{
-	urls: ["http://www.sitespeed.io", "http://www.sitespeed.io/faq/"],
-	browser: "chrome",
-	no: 17,
-	budget: {
-		timings: {
-			speedIndex: 1000,
-			headerTime: 800,
-			logoTime: 500,
-			domContentLoadedTime: 900
-		}
-	}
+Test your site against a [performance budget](http://timkadlec.com/2013/01/setting-a-performance-budget/). You can test your site against almost all data collected by sitespeed.io.
+
+Checkout the [example Gruntfile]() and budget looks something like this:
+
+```
+budget: {
+  "browsertime.pageSummary": [{
+    "metric": "statistics.timings.firstPaint.median",
+    "max": 1000,
+    "unit": "ms"
+  }, {
+    "metric": "statistics.visualMetrics.FirstVisualChange.median",
+    "max": 1000,
+    "unit": "ms"
+  }],
+  "coach.pageSummary": [{
+    "metric": "advice.performance.score",
+    "min": 75,
+    "unit": "percentage"
+  }, {
+    "metric": "advice.info.domElements",
+    "max": 200
+  }, {
+    "metric": "advice.info.domDepth.max",
+    "max": 10
+  }, {
+    "metric": "advice.info.iframes",
+    "max": 0
+  }, {
+    "metric": "advice.info.pageCookies.max",
+    "max": 5
+  }],
+  "pagexray.pageSummary": [{
+    "metric": "transferSize",
+    "max": 100000,
+    "unit": "bytes"
```

Comment:
We don't use the units, do you know where you saw that, I think we missed to remove it somewhere.